ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.kt

```diff
@@ -469,9 +470,11 @@ class TractActivity : BaseToolActivity<TractActivityBinding>(R.layout.tract_acti
     private var liveShareMenuObserver: Observer<Pair<State, State>>? = null
     private fun Menu.setupLiveShareMenuItemVisibility() {
         liveShareMenuObserver?.let { liveShareState.removeObserver(it) }
+
+        val liveShareItem = findItem(R.id.action_live_share_active)
+        liveShareItem?.loadAnimation(this@TractActivity, R.raw.anim_tract_live_share)
         liveShareMenuObserver = Observer<Pair<State, State>> { (publisherState, subscriberState) ->
-            findItem(R.id.action_live_share_active)?.isVisible =
-                publisherState == State.On || subscriberState == State.On
+            liveShareItem?.isVisible = publisherState == State.On || subscriberState == State.On
```

Comment:
The publisherState /subscriberState is referring to web socket logic for liveShare right? publisher being the host(not sure the correct term?), while the subscriber is the person who joins. And then the animation only shows if they are connected via live share?

Reply:
yes, the icon shows if the user is publishing or subscribing to a live share stream